### PR TITLE
Add Privacy page

### PR DESCRIPTION
**Pull request by NinaNinchy**
- create privacy.html
- add privacy in urls.py
- add privacy in views.py

### CODE REVIEW

**Comment by NinaNinchy** [comment]
Vidim, da sem naredilal pull request za vseh 11 commitov iz preteklosti ... hotela sem pa samo za te današnje :/

**Comment by livike** [comment]
Seems to be more efficient way to start locally with the current version from upstream. When I had the same situation I resolved this way:
- save your changes regarding the privacy page (by copy/pasting the code or by creating a branch with the current situation
- switch to the master and on the master branch:
- reset the local repository to the commit before your first commit in pull request (use git log to find the commit id, then git reset --hard <yourcommitid>
- make a pull from the upstream
- push the local repository to your github repository (using git push origin +master - the + before the master will force the push and clear all the old commits)
- when the versions of upstream and origin are identical, add your changes regarding the privacy page
- make your commits and  push them to the github repo
- make a new pull request (because the original one was deleted automatically by deleting the old commits)

I hope that this approach is "orthodox", but this is how I managed to make it work by researching online. 
Please experienced people, comment

**Comment by PettraV** [comment]
:+1:

File: web/templates/pages/privacy.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{% extends 'base.html' %}
+
+{% block title %}CodeCatz privacy{% endblock %}
+
+{% block content%}
```
Comment (ialja) [inline web/templates/pages/privacy.html]: Where's the content? Also, the content part should end with `{% endblock %}`

File: templates/login.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<html>
```
Comment (ialja) [inline templates/login.html]: Templates should extend the base template. Also, please make sure to sync your fork and place templates in the `web/templates/pages` folder.

File: TrackCat/settings.py [hunk 3]
```diff
@@ -95,7 +95,9 @@
 	os.path.join(BASE_DIR, "static"),
 )
 
+
 try:
 	from .settings_local import *
 except ImportError as e:
+
```
Comment (ialja) [inline TrackCat/settings.py]: What's with all these extra empty lines?

**Comment by ialja** [comment]
Closing this Pull request, as it got quite messy - the templates that were added don't fully work. Please revert to a previous commit or start fresh, but make sure you're synced with the main repository.

**Pull request closed without merging**